src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -267,6 +267,9 @@ void AdaptDispatch::_SetGraphicsOptionHelper(const DispatchTypes::GraphicsOption
     }
 }
 
+#pragma warning(push)
+#pragma warning(disable : 26497) // we do not want constexpr compilation because these always evaluate at runtime

```

Comment:
okay i get it - same as the above one. _we only give them runtime values_, but they could be constexpr if we passed a constexpr value. This is the thing you were talking about.

Reply:
yes. should I constexpr them anyway? It's really just allowing the compiler to do it. Not forcing it to.

Reply:
I like the idea of them being constexprable